src/main/java/com/actiontech/dble/rwsplit/RWSplitNonBlockingSession.java

```diff
@@ -32,6 +41,28 @@ public void execute(Boolean master, Callback callback) {
         execute(master, null, callback);
     }
 
+    public void execute(Boolean master, Callback callback, String sql) {
+        try {
+            SQLStatement statement = parseSQL(sql);
+            if (statement instanceof MySqlPrepareStatement) {
+                String simpleName = ((MySqlPrepareStatement) statement).getName().getSimpleName();
+                nameSet.add(simpleName);
+                rwSplitService.setTxStart(true);
```

Comment:
application point？
prepare has no practical relationship with tx; (doing so may lead to other aspects of influence)
If it's for some purpose, try something else.